pages/common/acme.sh.md

```diff
@@ -0,0 +1,36 @@
+# acme.sh
+
+> An ACME Shell script, an acme client alternative to certbot.
+> More information: <https://github.com/acmesh-official/acme.sh>
+
+- Issue a certificate using webroot mode:
+
+`acme.sh --issue --domain {{domain.tld}} --webroot {{/path/to/webroot}}`
```

Comment:
Does this path have to be asolute?

Reply:
@marchersimon  I believe that they have to be absolute, more info https://github.com/acmesh-official/acme.sh/issues/114